Sources/AnyCodable/AnyEncodable.swift

```diff
@@ -108,28 +108,24 @@ extension _AnyEncodable {
 
     #if canImport(Foundation)
     private func encode(nsnumber: NSNumber, into container: inout SingleValueEncodingContainer) throws {
-        switch Character(Unicode.Scalar(UInt8(nsnumber.objCType.pointee)))  {
```

Comment:
for applications where performance matters, this change should help make the code run faster.